vms/platformvm/state/metadata_codec.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"math"
+
+	"github.com/ava-labs/avalanchego/codec"
+	"github.com/ava-labs/avalanchego/codec/linearcodec"
+)
+
+const (
+	v0tag = "v0"
+	v0    = uint16(0)
+)
+
+var metadataCodec codec.Manager
+
+func init() {
+	c := linearcodec.New([]string{v0tag}, linearcodec.DefaultMaxSliceLength)
+	metadataCodec = codec.NewManager(math.MaxInt32)
```

Comment:
Is there a reason we use the default max slice length and max int? I feel like we normally either do both as the default or both as max int